tokio/src/time/clock.rs

```diff
@@ -65,6 +66,13 @@ cfg_test_util! {
         inner: Mutex<Inner>,
     }
 
+    // Used to track if the clock was ever paused. This is an optimization to
+    // avoid touching the mutex if `test-util` was accidentally enabled in
+    // release mode.
+    //
+    // A static is used so we can avoid accessing the thread-local as well.
+    static DID_PAUSE_CLOCK: AtomicBool = AtomicBool::new(false);
```

Comment:
This fails with loom. How about using the std atomic here and initializing it to true when `#[cfg(loom)]`?

Reply:
I just switched it to a std bool. Loom tests don't make sense with time right now anyway, so it shouldn't be hit.